frontend/src/App.js

```diff
@@ -1,12 +1,12 @@
 import CssBaseline from "@material-ui/core/CssBaseline";
 
-import Registration from "./Registration";
+import RegistrationContainer from "./containers/RegistrationContainer/RegistrationContainer";
```

Comment:
Whats the reason for the folder structure /containers/RegistrationContainer/RegistrationContainer?

Seems like a bit of dublication of information, maybe:
/containers/RegistrationContainer.jsx
or
/containers/registration/RegistrationContainer.jsx
?

Reply:
Maybe, but my idea was to keep each container/component together with a test file.